### PR TITLE
libp2p.io

### PR DESCRIPTION
**Pull request by RichardLitt**
This reworks the website using Hugo, which fixes some issues Jekyll was having with relative paths. There is still significant work to be done editing the copy.

### CODE REVIEW

**Comment by RichardLitt** [comment]
To do:

- [ ] Adjust the navbar. Could look something like this:
screen shot 2016-09-13 at 5 59 57 pm
It's going to be a bit difficult to make this work; check if the hugo theme has a native navbar you can work with.
- [ ] Write up sections
  - [ ] Blurb // communication
  - [ ] Spec
  - [ ] Implementations
  - [ ] Examples
  - [ ] Contribute :+1:
  - [ ] Notes and Discussions -- "Learn More" section
  - [ ] PubSub
  - [ ] Peer Routing & Content Routing
  - [ ] Roadmap (with hedging, as this is moving)
- [ ] Add the logo back in.
- [ ] Make sure that all of the spec links work fine.
- [ ] Write up a 'how to publish' note.

File: public/404.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        <link href="./index.xml" rel="alternate" type="application/rss+xml" title="404 Page not found &middot; libp2p" />
+      
+    
+    <meta name="generator" content="Hugo 0.16" />
+
+    <link rel="canonical" href="./404/" />
+
+    
+<div id="particles-js"></div>
+<script src="https://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>
```
Comment (lgierth) [inline public/404.html]: This should be part of the repo -- rule of thumb: no external JS/CSS/images/assets

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    
+    
+    <meta property="og:type" content="website" />
+    
+
+  <title>
+     libp2p
+  </title>
+
+    <meta name="description" content="make peer-to-peer great again" />
```
Comment (lgierth) [inline public/index.html]: :-1: for "make X great again"

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+    <meta name="HandheldFriendly" content="True" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="shortcut icon" href="images/favicon.ico">
+	  <link rel="apple-touch-icon" href="images/apple-touch-icon.png" />
+    
+    <link rel="stylesheet" type="text/css" href="./css/screen.css" />
+    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Merriweather:300,700,700italic,300italic|Open+Sans:700,400|Inconsolata" />
```
Comment (lgierth) [inline public/index.html]: Same here re: external assets -- I'm sure there are more instances of this

File: public/404.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                </a>
+            
+            &nbsp;
+            
+                <a class="bloglogo" href="https://twitter.com/IPFSBot">
+                    <span class="icon-twitter" style="color:white;font-size:2em"></span>
+                </a>
+            
+            <h1 class="page-title">libp2p</h1>
+            <h2 class="page-description">make peer-to-peer great again</h2>
```
Comment (lgierth) [inline public/404.html]: Same here about "make X great again"

Comment (RichardLitt) [inline public/404.html]: Yep. Agreed.

**Comment by RichardLitt** [comment]
This needs to be overhauled in more ways. It will probably just be closed, pending a further design from @diasdavid.

**Comment by RichardLitt** [comment]
@VictorBjelkholm Closing this.

**Pull request closed without merging**